docs/en/loading/Kafka-connector-starrocks.md

```diff
@@ -242,5 +242,5 @@ transforms.unwrap.delete.handling.mode
 
 In the above configurations, we specify `transforms=addfield,unwrap`.
 
-- The addfield transform is used to add the __op field to each record of Debezium CDC-formatted data to support the StarRocks primary key model table. If the StarRocks table is not a Primary Key table, you do not need to specify the addfield transform. The addfield transform class is com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord. It is included in the Kafka connector JAR file, so you do not need to manually install it.
+- The addfield transform is used to add the __op field to each record of Debezium CDC-formatted data to support the StarRocks primary key table. If the StarRocks table is not a Primary Key table, you do not need to specify the addfield transform. The addfield transform class is com.Starrocks.Kafka.Transforms.AddOpFieldForDebeziumRecord. It is included in the Kafka connector JAR file, so you do not need to manually install it.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/dbafff1c-3b46-4124-872c-684dbdb8ccbf)
-> Primary Key. See to the other docs involved.